apps/native/components/PropertyProfileComponent.tsx

```diff
@@ -1,25 +1,25 @@
-import React, { useEffect, useState } from "react";
-import UserPropertyForm from "./UserPropertyForm";
-import { db } from "../firebase";
-import { collection, addDoc } from "firebase/firestore";
+// PropertyProfileComponent.js
```

Comment:
Is that meant to be PropertyProfileComponent.tsx?

Reply:
Yes, I have corrected it now.